README.md

```diff
@@ -164,3 +165,9 @@ You can see the complete list on GitHub, but here we list those who contributed
 - Patrick Del Conte [@exaV](https://github.com/exaV)
 
 [1]: https://plugins.gradle.org/plugin/com.strumenta.antlr-kotlin
+
+## License
+
+Users of this project can choose which license to apply: either the Apache License 2 or the BSD 3-Clause License.
+
+The licenses are alternatively.
```

Comment:
Don't think this is needed, as the above sentence already states a consumer have the choice.

Reply:
Removing it